app/models/concerns/contents_list.rb

```diff
@@ -0,0 +1,32 @@
+module ContentsList
+  extend ActiveSupport::Concern
+
+  included do
```

Comment:
I don't think this actually needs to be in an included block, and we don't need to extend ActiveSupport::Concern - default module behaviour allows object method definitions (which this is)

Reply:
Actually, I wonder if this should just be a model - it appears to rely on the class it's included in having a body method/attribute, and that's the only thing that gets passed in. Perhaps one to discuss on Monday morning.